src/steps/os/windows.rs

```diff
@@ -42,11 +42,6 @@ pub fn run_winget(ctx: &ExecutionContext) -> Result<()> {
 
     print_separator("winget");
 
-    if !ctx.config().enable_winget() {
-        print_warning("Winget is disabled by default. Enable it by setting enable_winget=true in the [windows] section in the configuration.");
-        return Err(SkipStep(String::from("Winget is disabled by default")).into());
-    }
-
```

Comment:
> Remove the check for 'winget_enable' set to 'true'. On my Windows 10 and 11 machines, there are no issues with Winget anymore. As far as I remember, it was disabled by default because it was buggy back then.

If this tool will no longer have issues, you should also consider removing its [configuration entry](https://github.com/topgrade-rs/topgrade/blob/2e70d132d05e3bdd5f5b7dd41f741aef0e4b3a50/config.example.toml#L187-L188), and NOTE, remove an existing config entry is a breaking change, you should document it in [BREAKINGCHANGES_dev.md](https://github.com/topgrade-rs/topgrade/blob/main/CONTRIBUTING.md#breaking-changes)